### ISSUE TITLE
Avoid using api.store directly

### ISSUE THREAD

**Issue by noxify** [u12206050/gridsome-plugin-algolia#4]
While running `gridsome develop` I got the following warning:

```
 WARNING  Deprecation notices

Avoid using api.store directly. Use the actions in api.loadSource() instead.
./node_modules/gridsome-plugin-algolia/index.js:2:17   
```

Gridsome version: 0.7.9
Plugin Version: 2.1.2

**Comment by darrenmothersele** [comment]
It looks like this is complaining because of the store parameter in the function signature. I'm guessing it is triggering the deprecation warning because it uses the accessor for store as defined [here](https://github.com/gridsome/gridsome/pull/639/files#diff-7e63e1d1346d5309c34f0106205d6dd6R24).

But the store parameter is not actually used by the plugin, so can it be safetly removed?

**Comment by darrenmothersele** [comment]
I've submitted a pull request, or use 
`yarn add https://github.com/darrenmothersele/gridsome-plugin-algolia.git`

**Comment by u12206050** [comment]
Thanks. Merged and Published 2.1.4
